README.md

```diff
@@ -110,6 +110,33 @@ Steps to follow:
 
 This will verify the current context against the swagger file we downloaded and copied over manually
 
+### Example of Usage in CI with Github Actions
+```
+name: Sample CI Workflow
+# This workflow is triggered on pushes to the repository.
+on: [push]
+env:
+  KUBEPUG_VERSION: "v1.1.0"
+  HELM_VERSION: "v3.2.4"
+  K8S_TARGET_VERSION: "v1.16.0"
+
+jobs:
+ api-deprecations-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check-out repo
+        uses: actions/checkout@v2
+
+      - name: Install Helm and Kubepug binaries
+        run: |
+          mkdir -p ~/bin
+          curl -sSL https://github.com/rikatz/kubepug/releases/download/${KUBEPUG_VERSION}/kubepug_linux_amd64.tar.gz | tar xvfz - --overwrite -C ~/bin/
```

Comment:
What about using the latest version everytime? https://github.com/rikatz/kubepug/releases/latest/download/kubepug_linux_amd64.tar.gz

Reply:
done